server.go

```diff
@@ -298,16 +301,41 @@ func removeTagsHTTP(w http.ResponseWriter, r *http.Request) {
 
 // Set the target file's name
 func setImageNameHTTP(w http.ResponseWriter, r *http.Request) {
-	bytes, err := ioutil.ReadAll(r.Body)
-
+	err := r.ParseForm()
 	if err != nil {
 		sendError(w, 400, err)
```

Comment:
Missing return on error
```suggestion
		sendError(w, 400, err)
		return
```